app/views/projects/index.html.erb

```diff
@@ -69,6 +69,14 @@
                 <% if can_delete_project %>
                 <%= link_to 'Delete', project, method: :delete, data: { confirm: 'Are you sure?' } %>
                 <%end%>
+
+                  <% if !project.archived  %>
+                    <%= link_to 'projects/archive/' + project.id.to_s , :class => "btn btn-white btn-sm", :data=> { :confirm => "Are you sure?" }  do %>
```

Comment:
@dawitd can we follow consistent routing for actions. E.g. to edit we use projects/3/edit so it would be good to follow the same pattern for actions too.
Please update routing to projects/{id}/archive. Plus to avoid any unwanted archiving of projects, make sure to use POST method rather than GET.